website/docs/r/ecs_task_definition.html.markdown

```diff
@@ -196,6 +197,52 @@ Guide](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/cluster-query-
 * `properties` - (Required) The set of network configuration parameters to provide the Container Network Interface (CNI) plugin, specified a key-value mapping.
 * `type` - (Optional) The proxy type. The default value is `APPMESH`. The only supported value is `APPMESH`.
 
+#### Inference Accelerators Arguments
+
+* `device_name` - (Required) The Elastic Inference accelerator device name. The deviceName must also be referenced in a container definition as a ResourceRequirement.
+* `device_type` - (Required) The Elastic Inference accelerator type to use.
+
+##### Example Usage:
+
+```hcl
```

Comment:
This example will likely trigger CI failures for formatting once on master, will fix post-merge.